gms/impl/src/main/java/com/linkedin/metadata/resources/dataprocess/DataProcesses.java

```diff
@@ -0,0 +1,167 @@
+package com.linkedin.metadata.resources.dataprocess;
+
+import com.linkedin.common.urn.DataProcessUrn;
+import com.linkedin.dataprocess.DataProcess;
+import com.linkedin.dataprocess.DataProcessInfo;
+import com.linkedin.dataprocess.DataProcessKey;
+import com.linkedin.metadata.aspect.DataProcessAspect;
+import com.linkedin.metadata.dao.BaseLocalDAO;
+import com.linkedin.metadata.dao.BaseSearchDAO;
+import com.linkedin.metadata.dao.utils.ModelUtils;
+import com.linkedin.metadata.restli.BaseSearchableEntityResource;
+import com.linkedin.metadata.search.CorpUserInfoDocument;
+import com.linkedin.metadata.snapshot.DataProcessSnapshot;
+import com.linkedin.parseq.Task;
+import com.linkedin.restli.common.ComplexResourceKey;
+import com.linkedin.restli.common.EmptyRecord;
+import com.linkedin.restli.server.annotations.Action;
+import com.linkedin.restli.server.annotations.ActionParam;
+import com.linkedin.restli.server.annotations.Optional;
+import com.linkedin.restli.server.annotations.QueryParam;
+import com.linkedin.restli.server.annotations.RestLiCollection;
+import com.linkedin.restli.server.annotations.RestMethod;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static com.linkedin.metadata.restli.RestliConstants.*;
+
+@RestLiCollection(name = "dataProcesses", namespace = "com.linkedin.dataprocess", keyName = "dataprocess")
+public class DataProcesses extends BaseSearchableEntityResource<
+    // @formatter:off
+    DataProcessKey,
+    DataProcess,
+    DataProcessUrn,
+    DataProcessSnapshot,
+    DataProcessAspect,
+    CorpUserInfoDocument> {
```

Comment:
This should be DataProcessDocument. Please change it in the rest of the file as well.